src/target/cortexm.c

```diff
@@ -682,7 +682,7 @@ bool cortexm_probe(ADIv5_AP_t *ap)
 		PROBE(renesas_probe);
 		break;
 	case JEP106_MANUFACTURER_ARM:
-		if (t->part_id == 0x4c0) {        /* Cortex-M0+ ROM */
+		if (t->part_id == 0x4c0 || t->part_id == 0x4c1) { /* Cortex-M0+ ROM (0x4c1 is for some newer LPC11U6xx)*/
```

Comment:
Rather than having this in the same if block, it is preferred that the `t->part_id` is tested against `0x4c1U` in an `else if` and the probe call duplicated. Having to reorganise this later as and when new parts are added to the block will be less than idea, especially as this change don't make it clear to the reader that the block is safe to split.